proto/sdk/blocksdk/v1/blocksdk.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package sdk.blocksdk.v1;
+
+import "gogoproto/gogo.proto";
+import "amino/amino.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/skip-mev/block-sdk/x/blocksdk/types";
+
```

Comment:
nit: docs

Reply:
As a rule of thumb, we like to provide docs on our proto obj, if you write them in the form of [godocs](https://go.dev/blog/godoc), the documentation will appear in the generated objects